llvm/docs/LangRef.rst

```diff
@@ -15868,6 +15868,51 @@ The returned value is completely identical to the input except for the sign bit;
 in particular, if the input is a NaN, then the quiet/signaling bit and payload
 are perfectly preserved.
 
+.. _i_fminmax_family:
+
+'``llvm.min.*``' Intrinsics Comparation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+   :widths: 16 28 28 28
+
+   * - Operation
+     - minnum/maxnum
+     - minimum/maximum
+     - minimumnum/maximumnum
+
+   * - ``NUM vs qNaN``
+     - NUM, no exception
+     - qNaN, no exception
+     - qNaN, no exception
+
+   * - ``NUM vs sNaN``
+     - qNaN, invalid exception
+     - qNaN, invalid exception
+     - NUM, invalid exception
+
+   * - ``qNaN vs sNaN``
+     - qNaN, invalid exception
+     - qNaN, invalid exception
+     - qNaN, invalid exception
+
+   * - ``sNaN vs sNaN``
+     - qNaN, invalid exception
+     - qNaN, invalid exception
+     - qNaN, invalid exception
+
+   * - ``+0.0 vs -0.0``
+     - either one
+     - +0.0(max)/-0.0(min)
+     - +0.0(max)/-0.0(min)
+
+   * - ``NUM vs NUM``
+     - larger(max)/smaller(min)
+     - larger(max)/smaller(min)
+     - larger(max)/smaller(min)
```

Comment:
It feels like this section needs a callout on how the sNaN entries interact with our current NaN-handling policy.

Reply:
The current text still isn't covering the interaction of sNaN with LLVM's NaN-handling policy.

It's also not entirely accurate to say that invalid exceptions are raised when FP exceptions are unspecified without constrained intrinsics.

Reply:
Sorry, I guess that I am not full understanding "LLVM's NaN-handling policy".
As my understanding, for fmax*, we should fellow the libc's semantics.

Reply:
For constrained ones, I guess that we should add more details in another PR(s).